tools/automation/generators/relnoted/relnotetester-nohighlights.yml

```
@@ -0,0 +1,58 @@
+product: Postgres Distributed
+version: 5.6
+date: 15 October 2024
+intro: |
+ Reassuring text about this release
+relnotes:
+- relnote: SDW support with Streaming Transaction
+  component: bdr
+  component_version: 5.6
+  details: |
+    One of the main advantages of streaming is that the WAL sender sends the partial transaction before it commits, which 
+    reduces replication lag. Now, with streaming support, the WAL decoder does the same thing, but it streams to the LCRs
```

Since we have "segments" as noun and LCR as adjective/determinator, I think this should be singular.
```suggestion
    reduces replication lag. Now, with streaming support, the WAL decoder does the same thing, but it streams to the LCR
```

---

```suggestion
    reduces replication lag. Now, with streaming support, the WAL decoder does the same thing, but it streams to the LCR's
```